Opcodes/hrtferx.h

```diff
@@ -41,4 +41,5 @@ typedef struct {
   MYFLT         outl[BUF_LEN], outr[BUF_LEN];
   MYFLT         x[BUF_LEN], yl[BUF_LEN], yr[BUF_LEN];
   MYFLT         bl[FILT_LENm1], br[FILT_LENm1];
+  void *setup, *isetup;
```

Comment:
Note: we should get the formatting branch merged in sometime soon, there's a lot of inconsistencies in recent PRs